tool/example.dx.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:butterfly/butterfly.dart';
+
+class FooWidget extends StatelessWidget {
+  @override
+  Node build() {
+    var local = false;
+    var bar = 2;
+    var items = [1, 2, 3].map((x) => Dx('''"${x}"''')).toList();
+
+    return Dx('''
+      <MyComponent foo=${bar}>
+        "This is text"
```

Comment:
I'm tempted to suggest that we introduce a `<text>` tag and end the "how should we handle whitespace between tags" discussion :)